src/containers/OrdinalBins.js

```diff
@@ -0,0 +1,140 @@
+import React, { PureComponent } from 'react';
+import { compose, bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import color from 'color';
+import { getNodeLabelFunction, makeNetworkNodesForSubject, makeGetOrdinalValues, makeGetPromptVariable } from '../selectors/interface';
+import { actionCreators as networkActions } from '../ducks/modules/network';
+import { NodeList } from '../components/';
+import { MonitorDragSource } from '../behaviours/DragAndDrop';
+import { getCSSVariableAsString } from '../utils/CSSVariables';
+
+class OrdinalBins extends PureComponent {
+  static propTypes = {
+    activePromptVariable: PropTypes.string.isRequired,
+    getLabel: PropTypes.func.isRequired,
+    bins: PropTypes.array.isRequired,
+    prompt: PropTypes.object.isRequired,
+    stage: PropTypes.object.isRequired,
+    toggleNodeAttributes: PropTypes.func.isRequired,
+  };
+
+  static defaultProps = {
+    isDragging: false,
+    meta: {},
+  };
+
+  renderOrdinalBin = (bin, index) => {
+    const stageId = this.props.stage.id;
+    const promptId = this.props.prompt.id;
+    const promptColor = getCSSVariableAsString(`--${this.props.prompt.color}`) ? color(getCSSVariableAsString(`--${this.props.prompt.color}`)) : color(getCSSVariableAsString('--ord-color-seq-1'));
+    const backgroundColor = color(getCSSVariableAsString('--background'));
+    const missingValue = bin.value < 0;
+    const accepts = () => true;
+
+    const calculateAccentColor = () => {
```

Comment:
These methods should probably be moved outside of this function, currently they will be defined every time the render method is run.

I might be tempted to break this into it's own component actually.